EVEStandard/API/APIBase.cs

```diff
@@ -116,12 +116,12 @@ internal APIBase(string dataSource)
 
         protected static void checkAuth(AuthDTO auth, string scope)
         {
-            if (auth?.Character == null || auth.AccessToken == null)
+            if (auth.CharacterId == 0 || auth?.AccessToken.AccessToken == null)
```

Comment:
This will throw an exception if auth is null.

We've lost the null check by removing auth?.